### ISSUE TITLE
Music 0.3.12 doesn't play music in IE11

### ISSUE THREAD

**Issue by zubanst** [owncloud/music#546]
Hello,
Music 0.3.12. in OC 9.0.4 doesn't play music with IE.11. There is an error in .../aurora.js/src/device.coffe saying SCRIPT5022: No supported audio device found. Here is the piece of code throwing the error.
# 
# The AudioDevice class is responsible for interfacing with various audio
# APIs in browsers, and for keeping track of the current playback time
# based on the device hardware time and the play/pause/seek state
# 

EventEmitter = require './core/events'

class AudioDevice extends EventEmitter
    constructor: (@sampleRate, @channels) ->
        @playing = false
        @currentTime = 0
        @_lastTime = 0

```
start: ->
    return if @playing
    @playing = true

    @device ?= AudioDevice.create(@sampleRate, @channels)
    unless @device
        **throw new Error "No supported audio device found."**

    @_lastTime = @device.getDeviceTime()

    @_timer = setInterval @updateTime, 200
    @device.on 'refill', @refill = (buffer) =>
        @emit 'refill', buffer

stop: ->
    return unless @playing
    @playing = false

    @device.off 'refill', @refill
    clearInterval @_timer

destroy: ->
    @stop()
    @device.destroy()

seek: (@currentTime) ->
    @_lastTime = @device.getDeviceTime() if @playing
    @emit 'timeUpdate', @currentTime

updateTime: =>
    time = @device.getDeviceTime()
    @currentTime += (time - @_lastTime) / @device.sampleRate * 1000 | 0
    @_lastTime = time
    @emit 'timeUpdate', @currentTime

devices = []
@register: (device) ->
    devices.push(device)

@create: (sampleRate, channels) ->
    for device in devices when device.supported
        return new device(sampleRate, channels)

    return null
```

module.exports = AudioDevice

**Comment by zubanst** [comment]
I forgot to mention that there is no error in the logs.

**Comment by paulijar** [comment]
The Aurora.js playback engine does not seem to work on any Internet Explorer. The next release (probably 0.3.13) is using SoundManager2 engine instead of Aurora.js when possible (#535). With this change, .mp3-playback will work on IE9+. The .ogg and .flac files still cannot be played on IE because it does not contain suitable codecs.

**Comment by paulijar** [comment]
The issue was fixed in 0.3.13 and the IE's inability to play audio formats other than mp3 is addressed in the readme. Closing.
